packages/mdc-tabs/tab/mdc-tab.scss

```diff
@@ -95,10 +95,11 @@ $mdc-tab-with-icon-and-text-height: 72px;
 }
 
 .mdc-tab__indicator {
+  $mdc-tab-indicator-horizontal-padding: 2px;
```

Comment:
Not sure if this is confusing. I could just inline these values.

Reply:
Do we really need this as a variable? If so, I think it should go next to the others on line 26.

Reply:
Na we don't...I thought it might be clearer, but its not complicated math. I'll remove it